src/XMakeTasks/Resources/Strings.resx

```diff
@@ -2606,7 +2606,19 @@
   <data name="Copy.ExactlyOneTypeOfLink">
     <value>MSB3891: Both "{0}" and "{1}" were specified in the project file. Please choose one or the other.</value>    
   </data>
-  
+
+  <!--
+        MSB3901 - MSB3910   Task: Telemetry
+        If this bucket overflows, pls. contact 'vsppbdev'.
```

Comment:
I just copy/pasted what was there.  It occurs 59 times :(  Technically if it overflows, you have to make a new bucket.
Should I update them all?